packages/stylelint-config/package.json

```diff
@@ -4,15 +4,17 @@
   "license": "MIT",
   "main": "stylelint.config.js",
   "dependencies": {
-    "prettier": "^2.4.1",
     "style-search": "^0.1.0",
     "stylelint-config-css-modules": "^4.4.0",
     "stylelint-config-standard": "^36.0.0",
     "stylelint-declaration-block-no-ignored-properties": "^2.8.0",
-    "stylelint-order": "^6.0.4",
-    "stylelint-prettier": "^2.0.0"
+    "stylelint-order": "^6.0.4"
   },
   "peerDependencies": {
     "stylelint": "^16.2.1"
+  },
+  "devDependencies": {
+    "prettier": "^3.2.5",
+    "stylelint-prettier": "^5.0.0"
```

Comment:
https://github.com/prettier/stylelint-prettier/issues/298#issuecomment-1626867593

Reply:
These two packages need to be upgraded as a pair from everything that Taylor and I have seen in our research. The bumps here correspond to those found in https://github.com/GetJobber/Jobber/pull/45462/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519